### ISSUE TITLE
Analytics for Side (Left) Navigation

### ISSUE THREAD

**Issue by VictoriaSunNIH** [NCIOCPL/ncids#419]
Provide analytics for 
- [ ] Side navigation on desktop

Please note if there is a difference in what is captured for spanish sites (not shown in comp)

Invision Mockups:
Mockup: https://publicis.invisionapp.com/share/SY11BOIJQK4T#/454941204_Sidenav

Related Tickets:

- https://github.com/NCIOCPL/ncids/issues/312

**Comment by VictoriaSunNIH** [comment]
@VictoriaSunNIH - get mobile comps for side nav

**Comment by nicolecrom** [comment]
@VictoriaSunNIH @lburack @mworrest and any others in the know on the design for this item
We had a few questions before we can write requirements for this one
1. Is this on mobile and desktop?
2. Is there a button to open this menu or is it always on the page? Is this different on desktop v mobile
3. Can users navigate up or down the levels without navigating away from the page? Any other info on navigating we should note?
4. Which pages does this exist on? Is it basically on all "article" pages?

**Comment by VictoriaSunNIH** [comment]
@nicolecrom - as listed in the description, this is desktop only. Mobile does not have a side (left) navigation.  The left nav is always on the page, as long as it is an inner page (articles, press releases, any page that isnt a home/landing page or blog series/blog post page). 

I do not believe it allows you to navigate levels without navigation away from the page, since you click on the page to get to that section now (there is no expand/collapse) but looking at @mworrest or @lburack to confirm this one.

**Comment by nicolecrom** [comment]
@VictoriaSunNIH Thanks missed the desktop note somehow. So does this mean that the "section menu" on mobile is not being included in redesign or is that just a separate ticket?

**Comment by VictoriaSunNIH** [comment]
@nicolecrom analytics for mobile menu should have been captured in https://github.com/NCIOCPL/ncids/issues/415?  there is no "tongue" or separate section menu on mobile anymore, its all just one menu on mobile.

**Comment by nicolecrom** [comment]
@VictoriaSunNIH ok - thanks

**Comment by mworrest** [comment]
> @nicolecrom - as listed in the description, this is desktop only. Mobile does not have a side (left) navigation. The left nav is always on the page, as long as it is an inner page (articles, press releases, mini landing, dictionary pages, any page that isnt a home/landing page or blog series/blog post page, and some of the app modules like search results and CTS).
> 
> I do not believe it allows you to navigate levels without navigation away from the page, since you click on the page to get to that section now (there is no expand/collapse) but looking at @mworrest or @lburack to confirm this one.

Victoria is correct, we can't open other sections in the left nav without navigating to those pages.

**Comment by nicolecrom** [comment]
Here are the requirements for analytics for side nav
[Side Nav.docx](https://github.com/NCIOCPL/ncids/files/8765442/Side.Nav.docx)

**Comment by sarinapadilla** [comment]
I have added these ACs to https://github.com/NCIOCPL/cgov-digital-platform/issues/3344, so closing this ticket.
